code/API_definitions/location-subscriptions.yaml

```diff
@@ -0,0 +1,591 @@
+openapi: 3.0.3
+info:
+  title: Device location subscription event API
+  description: Service Enabling Network Function API which create, retrieve and delete subscriptions for events related to the location of a device
```

Comment:
As part of the API spec, we have to provide the API doc, as it is already done for location-verification.yaml. 
This can be completed in a separated PR once we agree on the content of this one.

Reply:
Yes, I saw API doc for location-verification.yaml. It means in this PR we do not need to change it? Shall we prepare a draft for this text and then discuss about content of it or how should be the process for that agreement?

Reply:
@chia2017 in this PR we should focus on the technical content and when this PR has been merged we can create a new PR with the documentation.